src/plugins/intel_gpu/include/intel_gpu/primitives/roi_align.hpp

```diff
@@ -24,6 +24,39 @@ struct roi_align : public primitive_base<roi_align> {
         Avg
     };
 
+    /// @brief Aligned mode for the @ref roi_align
+    enum AlignedMode {
+        Asymmetric,
+        Half_pixel_for_nn,
+        Half_pixel
+    };
+
+    /// @brief Constructs roi_align primitive.
+    /// @param id This primitive id.
+    /// @param inputs Inputs data primitive ids.
+    /// @param pooled_h Height of the ROI output feature map.
+    /// @param pooled_w Width of the ROI output feature map.
+    /// @param sampling_ratio Number of bins over height and width to use to calculate each output feature map element.
+    /// @param spatial_scale multiplicative spatial scale factor to translate ROI coordinates
+    /// from their input spatial scale to the scale used when pooling.
+    /// @param pooling_mode Method to perform pooling to produce output feature map elements.
+    roi_align(const primitive_id& id,
+              const std::vector<primitive_id>& inputs,
+              int pooled_h,
+              int pooled_w,
+              int sampling_ratio,
+              float spatial_scale,
+              PoolingMode pooling_mode,
+              const primitive_id& ext_prim_id = "",
+              const padding& output_padding = padding())
+        : primitive_base(id, inputs, ext_prim_id, output_padding),
+          pooled_h {pooled_h},
+          pooled_w {pooled_w},
+          sampling_ratio {sampling_ratio},
+          spatial_scale {spatial_scale},
+          pooling_mode {pooling_mode}
```

Comment:
what the default value of aligned_mode as you don't define it here? What the reason to have this constructor also?

Reply:
We need to have two constructors since we use both of them. Default value was added.

Reply:
Done